### DISCUSSION TITLE
Weather forecast stopped displaying

### DISCUSSION THREAD

**Discussion by Sawadee2u** [HASwitchPlate/HASPone#243] [Q&A]
I have three Haspone’s and they have been working for years. Recently after one of the Home Assistant update early in this year stopped displaying and the blueprint is not being triggered. I have it set to display on page 1 button 7 which is the standard location. For weather input I have Home, Pirate, and Accuweathee. Does not work on any. I have attribute set to condition with index 1 and display prefix to day. This worked until the update killed it.
Anyone had this issue?

**Comment by aderusha** [comment]
I'll have to re-work the entire blueprint, HA has removed the forecast attribute from the weather entities with the latest release.  Breaking changes are always fun.

**Comment by Sawadee2u** [comment]
Thanks. I hate breaking changes. The largest thorn in the backside with HA..Thanks again.* * * * * * * * * * * * * * * * * * * * * * * * *“Beer is Proof that God Loves Us and Wants Us to Be Happy"  Benjamin Franklin* * * * * * * * * * * * * * * * * * * * * * *On May 1, 2024, at 10:03 AM, Allen Derusha ***@***.***> wrote:﻿
I'll have to re-work the entire blueprint, HA has removed the forecast attribute from the weather entities with the latest release.  Breaking changes are always fun.
